neural_compressor/torch/quantization/algorithm_entry.py

```diff
@@ -64,7 +64,10 @@ def rtn_entry(
             "double_quant_group_size": quant_config.double_quant_group_size,
         }
 
-    model = rtn_quantize(model, weight_config=weight_config)
+    mode = kwargs.get("mode", "quantize")
```

Comment:
Recommend making the `mode` as a mandatory argument for the entry function.

Reply:
updated